guides/common/modules/proc_synchronizing-templates-with-a-local-directory-using-the-api.adoc

```diff
@@ -28,20 +28,24 @@ organizations:
 ----
 
 .Procedure
-. Create the directory where templates are stored and apply appropriate permissions and SELinux context:
+. In `/var/lib/foreman`, create a directory for storing templates:
 +
 [subs="+quotes"]
 ----
-# mkdir -p _/usr/share/templates_dir/_
-# chown foreman _/usr/share/templates_dir/_
-# chcon -t httpd_sys_rw_content_t _/usr/share/templates_dir/_ -R
+# mkdir /var/lib/foreman/_My_Templates_Dir_
 ----
-. Change the *Repo* setting on the *TemplateSync* tab to match the export directory `/usr/share/templates_dir/`.
++
+[NOTE]
+====
+You can place your templates to a custom directory outside `/var/lib/foreman`, but you have to set the `foreman_lib_t` SELinux label to that directory.
+====
+
+. Change the *Repo* setting on the *TemplateSync* tab to match the export directory `/var/lib/foreman/_My_Templates_Dir_/`.
 . Export templates from your {ProjectServer} to a local directory:
 +
 [options="nowrap", subs="verbatim,quotes,attributes"]
 ----
-$ curl -H "Accept:application/json" \
+# curl -H "Accept:application/json" \
 -H "Content-Type:application/json" \
 -u _login_:__password__ \
 -k https://_{foreman-example-com}/api/v2/templates/export \
```

Comment:
Just below this is `-X POST \` but I think that should be `-X POST`.

Reply:
Thanks, fixed.